mypyc/ir/rtypes.py

```diff
@@ -174,7 +175,7 @@ def __init__(self,
         self.is_unboxed = is_unboxed
         self._ctype = ctype
         self.is_refcounted = is_refcounted
-        if ctype in ('CPyTagged', 'Py_ssize_t'):
+        if ctype in ('CPyTagged', 'int32_t', 'int64_t'):
             self.c_undefined = 'CPY_INT_TAG'
```

Comment:
For low-level integers we actually don't have undefined values, since all values values are possible (at least in some contexts). We don't have a good way to represent this right now, so maybe add a TODO comment about this?